packages/template-block/src/helpers/rtePlugin.ts

```diff
@@ -0,0 +1,38 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    AlignCenterPlugin,
+    AlignJustifyPlugin,
+    AlignLeftPlugin,
+    AlignRightPlugin,
+    BoldPlugin,
+    ItalicPlugin,
+    PluginComposer,
+    ResetFormattingPlugin,
+    SoftBreakPlugin,
+    StrikethroughPlugin,
+    TextStylePlugin,
+    UnderlinePlugin,
+} from '@frontify/fondue';
+import { TextStylePluginsWithoutImage, TextStylesWithoutImage } from '@frontify/guideline-blocks-settings';
+
+export const getTitlePlugin = () => {
+    return new PluginComposer()
+        .setPlugin([new SoftBreakPlugin(), new TextStylePlugin({ textStyles: TextStylePluginsWithoutImage })])
+        .setPlugin([new BoldPlugin(), new ItalicPlugin(), new UnderlinePlugin(), new StrikethroughPlugin()])
+        .setPlugin([
+            new AlignLeftPlugin({
+                validTypes: TextStylesWithoutImage,
+            }),
+            new AlignCenterPlugin({
+                validTypes: TextStylesWithoutImage,
+            }),
+            new AlignRightPlugin({
+                validTypes: TextStylesWithoutImage,
+            }),
+            new AlignJustifyPlugin({
+                validTypes: TextStylesWithoutImage,
+            }),
+            new ResetFormattingPlugin(),
```

Comment:
```suggestion
            new ResetFormattingPlugin(),
            new AutoformatPlugin(),
```

Newly created Plugin for autoformatting

Reply:
Thanks!